build.gradle

```diff
@@ -32,6 +34,54 @@ compileJava {
 	options.release = 11
 }
 
+javadoc {
```

Comment:
What is the purpose of creating javadoc here?
I have no problem understanding `task buildDocs`, but not sure why it depends on javadoc.

Reply:
Because the purpose of the task is to build all the docs, which means the javadoc task need to run as well. The dependency forces that to happen

Reply:
👍